### PR TITLE
Fix window title that is not displayed

### PR DESCRIPTION
**Pull request by Huluti**
In current version, the titles aren't displayed. Here the fix!
I also removed an unuseful import.

### CODE REVIEW

File: pacman_mirrors/pacman_mirrors.py [hunk 1]
```diff
@@ -518,24 +518,17 @@ def write_interactive_mirrorlist(self):
         Write the mirrorlist file, the 'Custom' mirror file and modify
         the configuration file to use the 'Custom' file.
         """
-        from .pacman_mirrors_gui import chooseMirrors
+        import .pacman_mirrors_gui
```
Comment (Kirek) [inline pacman_mirrors/pacman_mirrors.py]: should be `from . import pacman_mirrors_gui`

**Comment by Kirek** [comment]
The are few functionality regression from the current code:
- It doesn't have anypagination when there are more than 10 mirrors, and the list can be easily bigger than the screen if you use `pacman-mirrros -i -c all`
- For selecting multiple mirrors you need to press ctrl ar mays, while in the previous one you just need to keep clicking the mirrors.
- The windows is not centered, you need to add something like win.set_position(Gtk.WindowPosition.CENTER)

**Comment by Huluti** [comment]
Oh sorry, I didn't want to push the last commit... just the two first.

**Comment by Kirek** [comment]
Thanks

**Pull request merged**